Casks/gsuite.rb

```diff
@@ -0,0 +1,11 @@
+cask 'g-desktop-suite' do
```

Comment:
This won’t work, because the token does not match the filename.

But what’s the point of adding the cask here, if you’ve submitted it to the main repo?